SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -549,6 +549,56 @@ FunctionDef {
 		^().putPairs(this.keyValuePairsFromArgs)
 	}
 
+
+	prMakePerformableEnvir {|argsArray=([]), keywordArgsEvent=(()), dropFirstN=0|
```

Comment:
maybe the extra argument `dropFirstN` could be made unnecessary if we would not write 
```
var argNamesNoThis = argNames.drop(dropFirstN);
```
but implement a method for both Method and FunctionDef
```
+ Method {
    argNamesForCall { ^argNames.drop(1) }
    defaultArgs { ^prototypeFrame.drop(1) }
}

+ FunctionDef {
    argNamesForCall { ^argNames }
// defaultArgs is already defined for FunctionDef
}
```

Then we just write:

```
var argNamesNoThis = this.argNamesForCall;
```